Workbooks/BETTER_MTD_Workbook.json

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
Need to update the Workbook metadata file WorkbooksMetadata.json  to include the new workbook. This is critical and add workbook metadata definition.

Reply:
Included it in the BETTER MTD workbook to WorkbooksMetadata.json and added the required logo and preview images